src/test/kotlin/io/openfuture/state/base/MongoRepositoryTests.kt

```diff
@@ -0,0 +1,33 @@
+package io.openfuture.state.base
+
+import io.openfuture.state.repository.MongoIndexCreator
+import org.junit.jupiter.api.BeforeEach
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.autoconfigure.mongo.embedded.EmbeddedMongoAutoConfiguration
+import org.springframework.boot.test.autoconfigure.data.mongo.DataMongoTest
+import org.springframework.data.mongodb.core.MongoTemplate
+import org.springframework.data.mongodb.core.ReactiveMongoTemplate
+import org.springframework.data.mongodb.core.convert.MongoConverter
+
+@DataMongoTest(excludeAutoConfiguration = [EmbeddedMongoAutoConfiguration::class])
+abstract class MongoRepositoryTests {
+
+    // @formatter:off
+    @Autowired
+    protected lateinit var mongoTemplate: MongoTemplate
```

Comment:
at the latest versions of spring boot this bean is missing

So, just remove it